src/netconfig.rs

```diff
@@ -28,26 +31,20 @@ pub fn read_netconfig(path: impl AsRef<Path>) -> Result<Option<Netconfig>, anyho
 }
 
 fn handle_netconfig_values() -> Result<Option<Netconfig>, anyhow::Error> {
+    let mut netconfig = Netconfig::default();
     if let Ok(dns_policy) = dotenv::var("NETCONFIG_DNS_POLICY") {
-        let dns_policies: Vec<&str> = dns_policy.split(' ').collect();
-        if dns_policies.len() > 1 {
-            return Err(anyhow::anyhow!(
-                "For NETCONFIG_DNS_POLICY only single policies are supported"
-            ));
-        }
-        let dns_policy = dns_policies[0];
-        match dns_policy {
-            "" => return Ok(None),
-            "STATIC" => (),
-            "auto" => (),
-            _ => {
-                return Err(anyhow::anyhow!(
-                    "For NETCONFIG_DNS_POLICY only \"STATIC\" and \"auto\" are supported"
-                ))
+        if dns_policy == "auto" {
+            netconfig.dns_policy = vec!["STATIC".to_string(), ".*".to_string()];
+        } else if !dns_policy.is_empty() {
+            if dns_policy.contains(&"STATIC_FALLBACK".to_string()) {
+                anyhow::bail!("NETCONFIG_DNS_POLICY \"STATIC_FALLBACK\" is not supported");
             }
+
+            let dns_policy = dns_policy.replace("?", "[0-9]");
+            let dns_policy = dns_policy.replace("*", ".*");
```

Comment:
What about `.` in the policy. I guess you would need to mark it as `\.`

Reply:
I mean for vlan interfaces called `eth0.11` that they do not match `eth0211` :clown_face: